README.md

```diff
@@ -2,71 +2,113 @@
 
 This example app uses [Knock](https://knock.app) to power cross channel notifications via email, an in-app feed, and Slack inside of a full-stack Node application, written in [Blitz.js](https://blitzjs.com/). It uses the [Knock Node SDK](https://github.com/knocklabs/knock-node) and [React in-app feed components](https://github.com/knocklabs/react-notification-feed).
 
-You can read more about this example app [in the Knock documentation](https://docs.knock.app/getting-started/example-app).
+You can read more about this example app [in the Knock documentation](https://docs.knock.app/getting-started/example-app#nodejs-example-app).
 
-## Running locally
+## Set up
 
-First, configure the environment variables properly (see the bottom of this README).
+### In the Knock dashboard
 
-Then run:
+[Sign up for a Knock account](https://dashboard.knock.app/signup) to configure channels and workflows for the app to work as expected. Once you've created your Knock account you will need to create channels and workflows to work with your application.
 
-```
-yarn blitz prisma migrate dev
-```
+1. Create an email channel and an in-app feed channel.
+   - You'll need to use an existing email provider like [Postmark](https://postmark.com/) to configure to your email channel in the Knock dashboard.
+2. A workflow in Knock with the key `welcome`. It should include an email step for your welcome email template.
+3. A workflow in Knock with the key `new-comment`. It should include: an in-app feed step and an email step. On these step's templates you can use the following variables which the example app is already configured to send as arguments on the workflow trigger call:
+   1. `comment_content`: The content of the created comment.
+   2. `asset_name`: The name of the commented asset.
+   3. `asset_url`: The url of the commented asset.
+   4. `project_name`: The name of the project that the comment asset belongs to.
+4. A workflow in Knock with the key `new-asset`. It should include: an in-app feed step and an email step. On these step's templates you can use the following variables which the example app is already configured to send as arguments on the workflow trigger call:
+   1. asset_url: The url of the commented asset.
+   2. project_name: The name of the project that the comment asset belongs to.
 
-Then, let's get the database populated with seed data by running:
+> **🚀 Tip: auto-create workflows with the CLI 🚀** <br> The example app repo contains a `knock/workflows` directory with the three workflows listed above. You can use the [Knock CLI](https://docs.knock.app/cli) to push these workflows into your Knock account using the `knock workflow push --all` command.
 
-```
-yarn blitz db seed
-```
+### On your machine
 
-Finally, we can start the development server by running:
+**1. In your terminal clone this project and install dependencies:**
```

Comment:
Maybe we should break these out into multiple steps (Clone, install deps, run postgres), and maybe we put run postgres right before the migrations step.